README.rst

```diff
@@ -100,8 +100,14 @@ There are a few optional keyword arguments that are useful only for S3 access.
 
   >>> smart_open.smart_open('s3://', host='s3.amazonaws.com')
   >>> smart_open.smart_open('s3://', profile_name='my-profile')
+  >>> smart_open.smart_open('s3://', s3_upload={
+      'ServerSideEncryption': 'AES256'
```

Comment:
One-liner please (like the other examples).

Reply:
fixed